controls/cis_rhel8.yml

```
@@ -0,0 +1,2300 @@
+    automated: yes
+    rules:
+      - file_permissions_home_dirs
+
+  # NEEDS RULE (for user ownership)
+  # https://github.com/ComplianceAsCode/content/issues/5507
+  - id: 6.2.8
+    title: Ensure users own their home directories (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_groupownership_home_directories
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5506
+  - id: 6.2.9
+    title: Ensure users' dot files are not group or world writable (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5505
+  - id: 6.2.10
+    title: Ensure no users have .forward files (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 6.2.11
+    title: Ensure no users have .netrc files (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - no_netrc_files
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5504
+  - id: 6.2.12
+    title: Ensure users' .netrc Files are not group or world accessible (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 6.2.13
+    title: Ensure no users have .rhosts files (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - no_rsh_trust_files
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5503
+  - id: 6.2.14
+    title: Ensure all groups in /etc/passwd exist in /etc/group (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5502
+  - id: 6.2.15
+    title: Ensure no duplicate UIDs exist (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5501
+  - id: 6.2.16
+    title: Ensure no duplicate GIDs exist (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 6.2.17
+    title: Ensure no duplicate user names exist (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - account_unique_name
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5500
+  - id: 6.2.18
+    title: Ensure no duplicate group names exist (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5499
+  - id: 6.2.19
+    title: Ensure shadow group is empty (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 6.2.20
+    title: Ensure shadow group is empty (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
```

This rule lacks checks and remediations. I would drop it or move it into related_rules.
```suggestion
    automated: no
```

---

Done